groups/bsl/bslstl/bslstl_variant.h

```diff
@@ -436,9 +437,8 @@ typename add_pointer<
                                        variant<HEAD, TAIL...> >::type>::type
 get_if(const variant<HEAD, TAIL...> *obj) BSLS_KEYWORD_NOEXCEPT;
     // Return a pointer to the alternative object at specified 'INDEX' in the
-    // specified 'obj'.  If '*obj==nulptr' or 'TYPE' is not the type of the
-    // currently active alternative, return 'nullptr'.  'TYPE' shall appear
-    // exactly once in the variant's list of alternatives.  'INDEX' shall be a
+    // specified 'obj'.  If 'obj==nulptr' or 'INDEX' is not the index of the
```

Comment:
"nullptr" is still misspelled
```suggestion
    // specified 'obj'.  If 'obj == nullptr' or 'INDEX' is not the index of the
```